### ISSUE TITLE
Add Unpack rgba to Logix node browser

### ISSUE THREAD

**Issue by shiftyscales** [Frooxius/NeosPublic#1161]
Reported by Kulza#8623, Unpack rgba is absent from the node browser despite Pack rgba, and (Un)Pack xyzw being present.

**Comment by TehTurk** [comment]
This actually exists, but as I recall it's associated as an overload.

**Comment by ProbablePrime** [comment]
That's right, Turk, Unpack XYZW will work, I think the ask here is to create a square in the node browser that explicitly says "Unpack RGBA" for clarity though.

**Comment by Frooxius** [comment]
Yes, this is a bit tricky, I'll need to add new mechanism. The Unpack RGBA is an overload of the "Unpack XYZW" nodes so it automatically gets grouped into a single item. Currently it's not supported to display multiple overloads explicitly.
